nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/NiFiServiceFacade.java

```diff
@@ -313,6 +313,11 @@
      */
     FlowConfigurationEntity getFlowConfiguration();
 
+    /**
+     * Gets the metrics for the flow.
+     */
+    void getFlowMetrics();
```

Comment:
minor: feels odd to call this a get when we don't return anything

Reply:
Changed to generateFlowMetrics() (I didn't use `create` as it appears to be meant for creating components via the facade)